docs/src/validator/get-started/setup-a-validator.md

```diff
@@ -0,0 +1,422 @@
+---
+title: Setup a Solana Validator
+sidebar_label: Setup a Validator
+---
+
+This is a guide for getting your validator setup on the Solana testnet cluster for the first time. Testnet is a Solana cluster that is used for performance testing of the software before the software is used on mainnet. Since testnet is stress tested daily, it is a good cluster to practice validator operations.
+
+Once you have a working validator on testnet, you will want to learn about [operational best practices](../best-practices/operations.md) in the next section. Although the guide is specific to testnet, it can be adapted to mainnet or devnet as well. Refer to the [clusters](../../clusters) section of the Solana docs to see example commands for each cluster.
+
+Now let's get started.
+
+## Open The Terminal Program
+
+To start this guide, you will be running commands on your trusted computer, not on the remote machine that you plan to use for validator operations. First, locate the terminal program on your _trusted computer_.
+
+- on Mac, you can search for the word _terminal_ in spotlight.
+- on Ubuntu, you can type `CTRL + Alt + T`.
+- on Windows, you will have to open the command prompt as an Administrator.
+
+## Install The Solana CLI Locally
+
+To create your validator vote account, you need to install the [Solana command line interface](../../cli.md) on your local computer.
+
+You can either use [Solana's Install Tool](../../cli/install-solana-cli-tools#use-solanas-install-tool) section from the Solana docs to install the CLI, or alternatively, you can also [build from source](../../cli/install-solana-cli-tools#build-from-source).
+
+> Building from source is a great option for those that want a more secure and potentially more performant executable.
+
+Once the Solana CLI is installed, you can return to this document once you are able to run the following command and get an answer on your terminal:
+
+```
+solana --version
+```
+
+You should see an output that looks similar to this (note your version number may be higher):
+
+```
+solana-cli 1.14.17 (src:b29a37cf; feat:3488713414)
+```
+
+Once you have successfully installed the cli, the next step is to change your config so that it is making requests to the `testnet` cluster:
+
+```
+solana config set --url https://api.testnet.solana.com
+```
+
+To verify that your config has change run:
+
+```
+solana config get
+```
+
+You should see a line that says: `RPC URL: https://api.testnet.solana.com`
+
+## Create Keys
+
+On your local computer, create the 3 keypairs that you will need to run your validator ([docs for reference](../../running-validator/validator-start#generate-identity)):
+
+```
+solana-keygen new -o validator-keypair.json
+```
+
+```
+solana-keygen new -o vote-account-keypair.json
+```
+
+```
+solana-keygen new -o authorized-withdrawer-keypair.json
```

Comment:
might be another place we can reccomend validators to use a ledger as their authorized withdrawer. Mainly think people may not realize that its possible and a great idea to do so.